docs/modules/integrations/pages/github-actions/run-appium-script.adoc

```diff
@@ -0,0 +1,15 @@
+= Run Appium script using GitHub Actions
+:navtitle: Run Appium script
+
+Learn how to utilize Kobiton automation testing with Appium script and GitHub Actions.
+
+== Before you start
+
+You'll need the following:
+
+* Kobiton username and API key. To obtain them, follow the instructions in xref:profile:manage-your-api-credentials.adoc#_get_an_api_key[this guide].
+* `EXECUTOR_USERNAME` and `EXECUTOR_PASSWORD`. The executor username and password are the same as the Kobiton username and API key.
+
+== Run the Appium script
+
+To run Appium script on Kobiton devices using GitHub Actions, follow the instruction on our link:https://github.com/marketplace/actions/kobiton-execute-appium-test[official GitHub Marketplace Actions for Appium script] page.
```

Comment:
Suggested: 
Follow the instructions posted on GitHub Marketplace for the link:https://github.com/marketplace/actions/kobiton-execute-appium-test[Kobiton Action for Execute Appium Test].

Reply:
@Hikkigaya can you work on this?

Reply:
Sure

Reply:
Updated